pkg/build/builder/docker.go

```diff
@@ -53,9 +56,15 @@ func (d *DockerBuilder) Build() error {
 	if err != nil {
 		return err
 	}
-	if err := fetchSource(buildDir, d.build, d.urlTimeout, os.Stdin, d.git); err != nil {
+	sourceInfo, err := fetchSource(buildDir, d.build, d.urlTimeout, os.Stdin, d.git)
+	if err != nil {
 		return err
 	}
+	if sourceInfo != nil {
+		if err := updateBuildRevision(d.client, d.build, sourceInfo); err != nil {
```

Comment:
Yup, it will. We can simply log a warning that we couldn't update the commit info instead of failing the build. This means that if for some other reason we can't update the build, we can have a build that doesn't have revision info in it. @bparees is that acceptable?